src/webview/notifications.ts

```diff
@@ -14,6 +14,27 @@ export class NotificationsHandler {
 
       const notificationId = uuidV4();
 
+      /*
+      parse the token digits from sms body, find "token" or "code" in options.body which reflect the sms content 
+      ---
+      Token: 03624 / SMS-Code = PIN Token
+      ---
+      Prüfcode 010313 für Microsoft-Authentifizierung verwenden.
+      ---
+      483133 is your GitHub authentication code. @github.com #483133
+      ---
+      eBay: Ihr Sicherheitscode lautet 080090. \nEr läuft in 15 Minuten ab. Geben Sie den Code nicht an andere weiter.
+      ---
+      PayPal: Ihr Sicherheitscode lautet: 989605. Geben Sie diesen Code nicht weiter.
+      */
+      const rawBody = options.body;
+      const { 0: token } = /\d{5,6}/.exec(options.body) || [];
+      if (token && (['token', 'code'].find(a => options.body.toLowerCase().includes(a)))) {
```

Comment:
For some senders in India, they use the word `SMS` in the body

Reply:
i added "sms" to the array